mlir/patches/use-ir-from-jax.patch

```diff
@@ -1,103 +1,136 @@
 --- ir.py
 +++ ir.py
-@@ -2,95 +2,5 @@
- #  See https://llvm.org/LICENSE.txt for license information.
- #  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
- 
+@@ -1,131 +1,2 @@
+-#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
Just to better understand, is it a copy paste of the ir generated by the Jax project?

Reply:
This is an autogenerated file when creating the python bindings. However it conflicts with the python bindings provided by JAX. Here I am removing the attribute builders that need to be registered. JAX includes these as well and re-registering them causes an error.

Reply:
Thank you for the details